options/locale/locale_es-ES.ini

```diff
@@ -326,7 +326,7 @@ hi_user_x=Hola <b>%s</b>,
 
 activate_account=Por favor, active su cuenta
 activate_account.title=%s, por favor activa tu cuenta
-activate_account.text_1=¡Hola <b>%[1]s</b>, gracias por registrarse en %[2]!
+activate_account.text_1=¡Hola <b>%[1]s</b>, gracias por registrarse en %[2]s!
```

Comment:
Weird again:
![image](https://user-images.githubusercontent.com/51889757/135355979-7b09d3a9-6433-41ad-a4b9-0231346a6244.png)

Reply:
Sometimes when I translate I looking through other languages variants and 🤷🏻‍♂️